Lib/test/test_cmd_line.py

```diff
@@ -924,7 +924,7 @@ def test_python_gil(self):
                 self.assertEqual(proc.stderr, '')
 
     def test_python_asyncio_debug(self):
-        code = "import asyncio; print(asyncio.get_event_loop().get_debug())"
+        code = "import asyncio; print(asyncio.new_event_loop().get_debug())"
```

Comment:
I think this needs a `with contextlib.closing(...` to prevent the test suite printing a ResourceWarning